src/schema/comments.ts

```diff
@@ -550,7 +550,7 @@ export const reportCommentReasons = new Map([
 // eslint-disable-next-line @typescript-eslint/no-explicit-any
 export const resolvers: IResolvers<any, Context> = {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
-  Query: traceResolverObject<any, any>({
+  Query: traceResolverObject<any, any, Context>({
```

Comment:
Added generic for `Context` to resolver types so for schemas I did not yet convert to strict TS I am just giving `Context` for backward compatibility.